### PR TITLE
rename font related arguments, replace underscore with dash

### PR DESCRIPTION
**Pull request by CAMOBAP**
- metanorma/metanorma-standoc#375

### CODE REVIEW

**Comment by CAMOBAP** [comment]
Tests passed https://github.com/metanorma/metanorma-cli/actions/runs/462262618, revering Gemfile

**Comment by ronaldtse** [comment]
Seems that 2.5 and 2.6 Ubuntu are failing?

**Comment by CAMOBAP** [comment]
@ronaldtse they will fail until we release metanorma and metanomr-standoc

**Review by ronaldtse — APPROVED** [review]

File: lib/metanorma/cli/command.rb [hunk 1]
```diff
@@ -57,12 +58,16 @@ def compile(file_name = nil)
       option :format, aliases: "-x", type: :string, desc: "Formats to generate"
       option "output-folder", aliases: "-w", required: true, desc: "Directory to save compiled files"
       option :coverpage, aliases: "-c", desc: "Liquid template"
+      option :"agree-to-terms", type: :boolean, desc: "Agree / Disagree with all third-party licensing terms presented (WARNING: do know what you are agreeing with!)"
+      option :"no-install-fonts", type: :boolean, desc: "Skip the font installation process"
+      option :"continue-without-fonts", type: :boolean, desc: "Continue processing even when fonts are missing"
```
[the remark this answers is not in the document]
Comment (ronaldtse) [reply lib/metanorma/cli/command.rb:63]: We also don't need `:"agree-to-terms"`, just `"agree-to-terms"` is okay

**Pull request merged**